src/lib.rs

```diff
@@ -123,6 +154,34 @@
 #![deny(missing_docs)]
 #![warn(missing_debug_implementations, rust_2018_idioms)]
 
+
+
+pub use hyper::Method;
+
+/// Error types.
+pub mod error;
+
+/// The long-running session future we spawn for multiplexing onto a running WebDriver instance.
+mod session;
+mod traits;
+
+pub use traits::{NewConnector, CapabilitiesExt};
+
+/// Type alias for a Client with a Rustls connector
+///
+#[cfg(feature = "rustls-tls")]
+pub type RustlsClient =  session::Client<hyper_rustls::HttpsConnector<hyper::client::HttpConnector>>;
+
+/// Type alias for a Client with an OpenSSL connector
+///
+#[cfg(feature = "openssl-tls")]
+pub type OpenSslClient = session::Client<hyper_tls::HttpsConnector<hyper::client::HttpConnector>>;
```

Comment:
I wonder, what if instead of having these aliases, we instead just provide the following `impl` blocks:
```rust
#[cfg(feature = "rustls-tls")]
impl Client<hyper_rustls::HttpsConnector<hyper::client::HttpConnector>> {
    pub fn new_rusttls() -> Self {
        Client::with_connector(hyper_rustls::HttpsConnector<hyper::client::HttpConnector>::new())
    }
}

#[cfg(feature = "openssl-tls")]
impl Client<hyper_tls::HttpsConnector<hyper::client::HttpConnector>> {
    pub fn new() -> Self {
        Client::with_connector(hyper_tls::HttpsConnector<hyper::client::HttpConnector>::new())
    }
}
```
That way we can also get rid of the `NewConnector` trait entirely.

Reply:
That would work also, but I think that the type aliases help the user choose the TLS connector without specifying `hyper` and `hyper_rustls` or `hyper_tls` as their dependencies so even without the `NewConnector` trait I still think the type aliases are useful.

Reply:
Ah, you mean if they store a `Client<...>` in a struct of some kind, and therefore name the type? That's a good point — let's keep them!